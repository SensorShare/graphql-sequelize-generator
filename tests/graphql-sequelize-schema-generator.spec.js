const graphqlSchemaGenerator = require('../src/graphql-sequelize-schema-generator')
const models = require('./models')

it('outputs the correct schema', () => {
  const schema = graphqlSchemaGenerator(models)
  expect(schema).toMatchSnapshot()
})
