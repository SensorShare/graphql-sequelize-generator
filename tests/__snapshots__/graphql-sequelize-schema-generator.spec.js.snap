// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`outputs the correct schema (build version) 1`] = `
"Object{
  query: GraphQLObjectType{
    name: \\"Root\\",
    description: undefined,
    isTypeOf: undefined,
    _typeConfig: Object{
      name: \\"Root\\",
      fields: Object{
        Departments: Object{
          type: GraphQLList{
            ofType: GraphQLObjectType{
              name: \\"Department\\",
              description: undefined,
              isTypeOf: undefined,
              _typeConfig: Object{
                name: \\"Department\\",
                fields: #function#
              }
            }
          },
          resolve: #function#
        },
        Groups: Object{
          type: GraphQLList{
            ofType: GraphQLObjectType{
              name: \\"Group\\",
              description: undefined,
              isTypeOf: undefined,
              _typeConfig: Object{
                name: \\"Group\\",
                fields: #function#
              }
            }
          },
          resolve: #function#
        },
        Torrents: Object{
          type: GraphQLList{
            ofType: GraphQLObjectType{
              name: \\"Torrent\\",
              description: undefined,
              isTypeOf: undefined,
              _typeConfig: Object{
                name: \\"Torrent\\",
                fields: #function#
              }
            }
          },
          resolve: #function#
        },
        Users: Object{
          type: GraphQLList{
            ofType: GraphQLObjectType{
              name: \\"User\\",
              description: undefined,
              isTypeOf: undefined,
              _typeConfig: Object{
                name: \\"User\\",
                fields: #function#
              }
            }
          },
          resolve: #function#
        }
      }
    }
  }
}"
`;

exports[`outputs the correct schema 1`] = `
"Object{
  query: GraphQLObjectType{
    name: \\"Root\\",
    description: undefined,
    isTypeOf: undefined,
    _typeConfig: Object{
      name: \\"Root\\",
      fields: Object{
        Departments: Object{
          type: GraphQLList{
            ofType: GraphQLObjectType{
              name: \\"Department\\",
              description: undefined,
              isTypeOf: undefined,
              _typeConfig: Object{
                name: \\"Department\\",
                fields: #function#
              }
            }
          },
          resolve: #function#
        },
        Groups: Object{
          type: GraphQLList{
            ofType: GraphQLObjectType{
              name: \\"Group\\",
              description: undefined,
              isTypeOf: undefined,
              _typeConfig: Object{
                name: \\"Group\\",
                fields: #function#
              }
            }
          },
          resolve: #function#
        },
        Torrents: Object{
          type: GraphQLList{
            ofType: GraphQLObjectType{
              name: \\"Torrent\\",
              description: undefined,
              isTypeOf: undefined,
              _typeConfig: Object{
                name: \\"Torrent\\",
                fields: #function#
              }
            }
          },
          resolve: #function#
        },
        Users: Object{
          type: GraphQLList{
            ofType: GraphQLObjectType{
              name: \\"User\\",
              description: undefined,
              isTypeOf: undefined,
              _typeConfig: Object{
                name: \\"User\\",
                fields: #function#
              }
            }
          },
          resolve: #function#
        }
      }
    }
  }
}"
`;
